packages/cli/src/handlers/setProject.ts

```diff
@@ -49,6 +49,39 @@ export const setProjectInteractively = async () => {
     return answers.project;
 };
 
+export const setProjectByName = async (projectName: string): Promise<void> => {
+    const projects = await lightdashApi<OrganizationProject[]>({
+        method: 'GET',
+        url: `/api/v1/org/projects`,
+        body: undefined,
+    });
+
+    GlobalState.debug(
+        `> Set project returned response: ${JSON.stringify(projects)}`,
+    );
+
+    if (projects.length === 0) throw new Error('No projects found.');
+
+    const selectedProject = projects.find(
+        (project) => project.name === projectName,
```

Comment:
project names are not unique so there can be multiple projects with the same name. should we change it to uuid?

we have a ticket about project name uniqueness: https://github.com/lightdash/lightdash/issues/6828

Reply:
I think #6828 is on the right track: project names should be unique. If that is put in, then I think name makes more sense. Imo, using uuid in the interface is cumbersome, because there's currently no way to fetch the uuid's. 

Whereas, projects can often be created using a unique name. In our flow for example, which inspired this feature, we are doing `lightdash start-preview --name $CI_COMMIT_REF` to create/or refresh a uniquely named preview.

Reply:
hi @tnightengale, I see, yeah, that makes sense.

I was thinking that we might need to make slight changes in the PR so it's consistent with what we have now. for example:

we already have `lightdash config set-project` which launches interactive project selector.

adding a `--name` there will be the best option which will not run interactive project selector and picks project by its name

`lightdash config set-project --name [project name]`

what do you think?

Reply:
@IrakliJani Yes, I agree ✅ I'll make the change.

Reply:
sounds great, thanks @tnightengale 🙌